numpy/typing/tests/data/reveal/arithmetic.py

```diff
@@ -126,7 +126,7 @@
 reveal_type(c16 + b)  # E: numpy.complexfloating[numpy.typing._64Bit, numpy.typing._64Bit]
 reveal_type(c16 + c)  # E: numpy.complexfloating[numpy.typing._64Bit, numpy.typing._64Bit]
 reveal_type(c16 + f)  # E: numpy.complexfloating[numpy.typing._64Bit, numpy.typing._64Bit]
-reveal_type(c16 + i)  # E: numpy.complexfloating[Any, Any]
+reveal_type(c16 + i)  # E: numpy.complexfloating[numpy.typing._
```

Comment:
What's going on here? Looks like invalid syntax, but repeated below so probably on purpose.

Reply:
So the comment on the right contains the type as expected by mypy. The problem is that the value is platform-dependent while the comment is... just a normal static comment. The workaround I employed here is the use the fact that the `reveal` tests actually search for substring matches, _i.e._ it checkes whether `comment is in expected_value`.

Reply:
Makes sense, thanks for the explanation.

Reply:
Xref https://github.com/numpy/numpy/issues/17617 as it is very much relevant for this particular issue.

Reply:
Perhaps worth a comment on the line above,
```suggestion
# note this comment is deliberate truncated as the result varies by platform
reveal_type(c16 + i)  # E: numpy.complexfloating[numpy.typing._
```

Reply:
> Perhaps worth a comment on the line above,

Good call, done as of https://github.com/numpy/numpy/pull/17843/commits/b1b047e0b977463ef97e54bf5a9685b39ae3dd93.